src/ModelerApp.vue

```diff
@@ -9,19 +9,19 @@
 
         <div class="ml-auto">
           <b-btn variant="secondary" size="sm" v-b-modal="'uploadmodal'" class="mr-2">
-            <i class="fas fa-upload mr-1" />
+            <i class="fas fa-upload mr-1"/>
             {{ $t('Upload XML') }}
           </b-btn>
-          <b-btn variant="secondary" size="sm" data-test="downloadXMLBtn" @click="download">
-            <i class="fas fa-download mr-1" />
-            {{ $t('Download XML') }}
-          </b-btn>
         </div>
       </b-card-header>
       <b-card-body class="overflow-hidden position-relative p-0 vh-100">
-        <modeler ref="modeler" @validate="validationErrors = $event" @warnings="warnings = $event" />
+        <modeler @set-xml-manager="xmlManager = $event" @validate="validationErrors = $event" @warnings="warnings = $event"/>
```

Comment:
This is a pattern I've used before in react: it gets around having to use a ref to reach into the component's methods by emitting a "controller" object that gives the parent control over things in the child component.